README.md

```diff
@@ -324,21 +321,21 @@ to the value obtained from the model training phase.
 For convenience, we can encapsulate our DML invocations in a single script:
 
 	#!/bin/bash
-	
+
 	./bin/systemml ./scripts/datagen/genLinearRegressionData.dml -nvargs numSamples=1000 numFeatures=50 maxFeatureValue=5 maxWeight=5 addNoise=FALSE b=0 sparsity=0.7 output=linRegData.csv format=csv perc=0.5
-	
+
 	./bin/systemml ./scripts/utils/sample.dml -nvargs X=linRegData.csv sv=perc.csv O=linRegDataParts ofmt=csv
-	
+
 	./bin/systemml ./scripts/utils/splitXY.dml -nvargs X=linRegDataParts/1 y=51 OX=linRegData.train.data.csv OY=linRegData.train.labels.csv ofmt=csv
-	
+
 	./bin/systemml ./scripts/utils/splitXY.dml -nvargs X=linRegDataParts/2 y=51 OX=linRegData.test.data.csv OY=linRegData.test.labels.csv ofmt=csv
-	
+
 	./bin/systemml ./scripts/algorithms/LinearRegDS.dml -nvargs X=linRegData.train.data.csv Y=linRegData.train.labels.csv B=betas.csv fmt=csv
-	
+
 	./bin/systemml ./scripts/algorithms/GLM-predict.dml -nvargs X=linRegData.test.data.csv Y=linRegData.test.labels.csv B=betas.csv fmt=csv
 
 
-In this example, we've seen a small part of the capabilities of SystemML. For more detailed information, please 
+In this example, we've seen a small part of the capabilities of SystemML. For more detailed information, please
```

Comment:
Let's add a "conclusion header" here:

### Conclusion

In this example, we've seen a small part...